ligo/stablecoin/actions.ligo

```diff
@@ -13,13 +13,101 @@
  * Helpers
  *)
 
+(*
+ * Helper function used to reduce
+ *  `if (condition) then failwith (message) else skip;`
+ * appearances.
+ *)
+function fail_on
+  ( const condition : bool
+  ; const message   : string
+  ) : unit is if condition then failwith (message) else unit
+
+(*
+ * Authorizes current contract owner and fails otherwise.
+ *)
+function authorize_owner
+  ( const store : storage
+  ) : unit is
+  fail_on
+    ( Tezos.sender =/= store.roles.owner
+    , "NOT_OWNER"
+    )
+
+(*
+ * Ensures that sender is current pending owner
+ *)
+function authorize_pending_owner
+  ( const store : storage
+  ) : unit is case store.roles.pending_owner of
+    Some (pending_owner) -> fail_on
+      ( Tezos.sender =/= pending_owner
+      , "Sender is not current pending owner"
```

Comment:
Why the difference in fail message formats? ie "NOT_PENDING_OWNER" and "Sender is not current pending owner" ?
Also shouldn't the former say "Contract not in transfer" or something like that?

Reply:
Oh, have missed it, thanks